tests/component-tests/orator/js/OratorTests-Utils.js

```diff
@@ -125,6 +125,16 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
      * Assertions
      *******************************************************************************/
 
+    // assert that the element is actually in the document, not detached
+    fluid.tests.orator.assertNodeInDOM = function (msg, elm) {
```

Comment:
Functions at line 129 & 135 are pretty much the same. It might be simpler to define just one `isNodeInDOM` function with `$.contains(document, fluid.unwrap(elm));`. This function returns a boolean that can be verified in caller functions.

Reply:
I was trying to keep some symmetry with the `assertNodeExists` and `assertNodeNotExists` functions from jqUnit-browser.js

https://github.com/fluid-project/infusion/blob/master/tests/test-core/jqUnit/js/jqUnit-browser.js#L45-L51 

What do you think?

Reply:
I'm not a big fan of writing duplicate code to keep symmetry with somewhere else, but this is your choice and I respect. ;)

Let's keep as is.